activerecord/lib/active_record/connection_adapters/postgresql/schema_statements.rb

```diff
@@ -655,14 +655,10 @@ def unique_keys(table_name)
           SQL
 
           unique_info.map do |row|
-            deferrable = extract_constraint_deferrable(row["condeferrable"], row["condeferred"])
+            conkey = row["conkey"].tr("{}", "").split(",").map(&:to_i)
```

Comment:
```suggestion
            conkey = row["conkey"].delete("{}").split(",").map(&:to_i)
```

Reply:
I prefer String#delete here for some minor readability.

Reply:
fixed and force pushed 👍